clients/go/test/license_test.go

```diff
@@ -40,8 +40,7 @@ const license = `// Copyright © 2018 Camunda Services GmbH (info@camunda.com)
 
 var skip = map[string]bool{
 	// These files are generated.
```

Comment:
What if we turn this into a "skipList" with the prefixes of files/dirs to ignore? Then we wouldn't need to add a new if clause for every directory we want to ignore and we're not actually using this map as a map. Something like:

```suggestion
var skipList = []string{
  // These files are generated.
  "../pkg/pb/gateway.pb.go",
  "../internal/mock_pb/",
  "../vendor/",
}

func TestLicense(t *testing.T) {
	err := filepath.Walk("..", func(path string, fi os.FileInfo, err error) error {
    for _, skip := range skipList {
      if strings.HasPrefix(path, skip) {
        return nil
      }
    }
```

Reply:
Sounds good :+1: